README.md

```diff
@@ -0,0 +1,158 @@
+
+  <h3><b>Hello-Microverse Readme</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
```

Comment:
[OPTIONAL] As I mentioned above :point_up_2: please don't remove any section which is not an optional section but you remove the `install`, `Run tests` and `Deployment` sections so kindly add them :slightly_smiling_face: